### PR TITLE
Result: Add `additional_info` property

### PR DESCRIPTION
**Pull request by sils1297**
Fixes https://github.com/coala-analyzer/coala/issues/1935

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack 53efe26

File: coalib/bearlib/abstractions/Lint.py [hunk 1]
```diff
@@ -217,7 +217,7 @@ def _check_command(cls, executable, command: (list, tuple, None), fail_msg):
             if command is None:
                 return True  # when there are no prerequisites
             try:
-                check_call(command)
+                check_call(command, stdout=DEVNULL, stderr=DEVNULL)
```
Comment (Adrianzatreanu) [inline coalib/bearlib/abstractions/Lint.py]: I was using ``os.devnull`` . what's the difference between those?

**Comment by Adrianzatreanu** [comment]
ack fbdc046

File: coalib/bearlib/abstractions/Lint.py (hunk 1 above)
Comment (sils1297) [inline coalib/bearlib/abstractions/Lint.py]: this one's meant for being used like this, it's subprocess.DEVNULL after all

Comment (sils1297) [inline coalib/bearlib/abstractions/Lint.py]: see https://docs.python.org/3/library/subprocess.html?highlight=subprocess.devnull#subprocess.DEVNULL

Comment (sils1297) [inline coalib/bearlib/abstractions/Lint.py]: it's the same :)

**Comment by Adrianzatreanu** [comment]
ack b9bc75c

**Comment by Adrianzatreanu** [comment]
ack b9b57c5

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**